dbms/src/Common/ThreadStatus.cpp

```diff
@@ -4,6 +4,7 @@
 #include <Common/Exception.h>
 #include <Common/ThreadProfileEvents.h>
 #include <Common/TaskStatsInfoGetter.h>
+#include <Common/QueryProfiler.h>
```

Comment:
Why needed?

Reply:
`ThreadStatus` has `query_profiler_real` and `query_profiler_cpu` fields which will have incomplete type without this include

Reply:
Ok.